src/dotnetTemplate/Microsoft.PowerShell.Standard.Module.Template/Microsoft.PowerShell.Standard.Module.Template/.template.config/template.json

```diff
@@ -16,18 +16,18 @@
     "PowerShellStandardVersion":{
       "type": "parameter",
       "datatype":"choice",
-      "defaultValue": "5.1.0-preview-06",
+      "defaultValue": "5.1.0",
       "choices": [
         {
-          "choice": "5.1.0-preview-06",
+          "choice": "5.1.0",
           "description": "PowerShell Standard 5.1"
         },
         {
           "choice": "3.0.0-preview-02",
           "description": "PowerShell Standard 3.0"
         }
       ],
-      "replaces": "5.1.0-preview-06"
+      "replaces": "5.1.0"
```

Comment:
I'm not sure what this does exactly but figured it had to be changed to remove the `-preview-06` ref.

Reply:
this is the right call - basically, it will replace instances of what's in `replaces` with the `choice` based on what flags are used when `dotnet new` is called.